node/src/main.rs

```diff
@@ -146,6 +148,17 @@ async fn main() {
         std::process::exit(0);
     }
 
+    let subgraph_config: SubgraphPerfRules = match env_vars.performance_config_file {
+        Some(ref path) => {
+            info!(logger, "Reading subgraph configuration file `{}`", path);
+            toml::from_str::<SubgraphConfigSection>(&read_to_string(path).unwrap())
+                .unwrap()
+                .try_into()
+                .unwrap()
```

Comment:
This would be better as a method on `SubgraphConfigSection` - `main` doesn't need to know that this is TOML.

Also, there's a lot of unwrapping here. What do error messages look like when there's a syntax error in the file or when the file doesn't even exist? It's important that the error messages give users enough information to fix their mistakes.